python/rmm/_lib/device_buffer.pxd

```diff
@@ -14,12 +15,19 @@ cdef extern from "rmm/device_buffer.hpp" namespace "rmm" nogil:
         size_t size()
         size_t capacity()
 
+    void copy_to_host(const device_buffer& db, void* hb) except *
+    void copy_to_host(const device_buffer& db,
+                      void* hb,
+                      cudaStream_t stream) except *
+
 cdef class DeviceBuffer:
     cdef unique_ptr[device_buffer] c_obj
 
     @staticmethod
     cdef DeviceBuffer c_from_unique_ptr(unique_ptr[device_buffer] ptr)
 
+    cpdef bytes tobytes(self, uintptr_t stream=*)
```

Comment:
Is `uintptr_t` to allow for implicit conversion from Python?

Reply:
Yep. Just learned that myself too 🙂

Reply:
We might want to start using this in more Cython signatures to clarify where we mean something is a pointer as well as facilitate simpler casting.